### PR TITLE
Consolidated .feature, .call-out class names to .feature.

### PR DESCRIPTION
**Pull request by ghost**
Closes department-of-veterans-affairs/vets.gov-team#1806

- Changed class='call-out' to class='feature' in markup.

### CODE REVIEW

**Comment by tiffanybbrown-va** [comment]
The big change is in `_va.scss`. The other changes just update to use the `.feature` class name.

**Comment by tiffanybbrown-va** [comment]
A pretty straightforward change, but a big commit because of the content-change.

File: content/pages/404.md
Comment (melwoodard) [inline content/pages/404.md]: @tiffanybbrown-va this broke the layout on the 404 page a bit - I'm seeing a previously hidden "Search:" label, and the button is off-center

vets_gov__vets_gov

Comment (melwoodard) [inline content/pages/404.md]: @tiffanybbrown-va this broke the layout on the 404 page a bit - I'm seeing a previously hidden "Search:" label, and the button is off-center

vets_gov__vets_gov

**Review by melwoodard — CHANGES_REQUESTED** [review]

File: content/pages/404.md
Comment (tiffanybbrown-va) [inline content/pages/404.md]: Yeah ... that's only part of the problem, it seems. I think I got a little aggressive with some refactoring to remove CSS

**Comment by tiffanybbrown-va** [comment]
Search page now looks like this:

screen shot 2017-03-27 at 9 47 17 am

**Review by melwoodard — APPROVED** [review]
404 looks much better. Thanks!

**Comment by rroueche** [comment]
Sorry, I'm holding off bc I need to evaluate some content we had in progress (burials and education) to see if we need to update the classes there. Once I update those, I'll confirm here that this is good to go!

**Comment by tiffanybbrown-va** [comment]
Fixed a bunch of merge conflicts in this PR. I did a once-through on the markup and what-not, but it's entirely possible that I missed a broken page. Please put another set of eyes on it.

**Comment by melwoodard** [comment]
Looking through the pages affected by the merge conflict resolution, they all look right to me. Going over a handful of pages from each section again, also looks good.

**Comment by jcmeloni-usds** [comment]
Coooool.  Want to merge now or let @rroueche weigh in too?  Thanks for dealing w/ the conflicts.

**Comment by rroueche** [comment]
@jcmeloni-usds and @tiffanybbrown-va would you mind holding off? We have the burials branch merging and going live today, and then I'll want to comb through it to make sure that .feature class is correct throughout, and then merge this one. I'm also totally open to discuss a different way forward if there's a better one - I'm just trying to avoid any possible complications w/the burials launch today.

**Comment by jcmeloni-usds** [comment]
@rroueche totally following your lead on this -- whatever makes everyone most comfortable.

**Comment by rroueche** [comment]
Ok, thanks @jcmeloni-usds please hold for now then - and I should be able to do my comb-through tonight after deployment, and cc you both either tonight or tmrw morning to move fwd!

**Comment by tiffanybbrown-va** [comment]
I think this also closes department-of-veterans-affairs/vets.gov-team#613.

**Review by rroueche — APPROVED** [review]
Ok all good!

**Pull request merged**
